Source/Presentation/PresentationController.swift

```diff
@@ -3,13 +3,33 @@ import UIKit
 class PresentationController: UIPresentationController {
 
     private let dimmingView = UIView()
+    private lazy var gradient: CAGradientLayer = {
+        let gradient = CAGradientLayer()
+        gradient.type = .axial
+        gradient.locations = [0.55, 0.85]
+        gradient.startPoint = CGPoint(x: 0.5, y: 0.0)
+        gradient.endPoint = CGPoint(x: 0.5, y: 1.0)
+        return gradient
+    }()
 
-    init(presentedViewController: UIViewController,
-         presenting presentingViewController: UIViewController?,
-         dimmingViewColor: UIColor)
-    {
+    init(
+        presentedViewController: UIViewController,
+        presenting presentingViewController: UIViewController?,
+        dimmingViewColor: UIColor,
+        backgroundStyle: BackgroundStyle
+    ) {
         super.init(presentedViewController: presentedViewController, presenting: presentingViewController)
         self.dimmingView.backgroundColor = dimmingViewColor
+
+        switch backgroundStyle {
+        case .plain:
+            dimmingView.backgroundColor = dimmingViewColor
```

Comment:
looks like the same assignment is in the line 22

Reply:
Yes, fixed it